### ISSUE TITLE
Published Guide not Working

### ISSUE THREAD

**Issue by proubatsis** [OpenLiberty/guide-maven-multimodules#34]
- Running through the guide as intended does not work (published/master and multipane)
- JDK 11 doesn’t work (published/master and multipane)
  - This issue exists in both the published and multipane guide, it is because the following properties are set in the parent pom and not the ear pom. Since adding the `<parent/>` tags comes later in the guide, building the jar early on fails.
```
<maven.compiler.source>1.8</maven.compiler.source>
<maven.compiler.target>1.8</maven.compiler.target>
```
- Building modules individually doesn’t work from finish directory — tests fail due to 404 error (published/master and multipane)
   - Working from start directory, the `liberty:start-server` goal shows errors (port properties are null), and making it work by providing the ports via command line shows context root not found when opening the URL.

**Comment by evelinec** [comment]
I traced back the changes in the ear/pom.xml file and found that the `maven-ear-plugin` version was upgraded from `2.10.1` to `3.0.1` in https://github.com/OpenLiberty/guide-maven-multimodules/pull/20/commits/516a86aeaf0df1b47149df40d85cddd15e5fb659. I reverted the changes from this PR and the published/master code is working. With this reversion, JDK11 is working as well. We need further confirmation as of why the latest `maven-ear-plugin` doesn't work with our application. 

As for the multipane, I believe some changes were made in the `start/pom.xml` and the`ear/pom.xml` in attempt to make used of the Liberty parent pom.. however, it wasn't done properly, so the problem mentioned in the description is seen. Again, reverting all these changes along with the `maven-ear-plugin` version reversion, resolves the problem.
